nemo_text_processing/text_normalization/es/graph_utils.py

```diff
@@ -184,4 +194,21 @@ def _load_roman(file: str):
         )
     ).optimize()
 
-    return graph @ fst
+    graph_upper = (
```

Comment:
Could you explain the need for the change here?

Reply:
The current ES TN model tags the name `Li` as an ordinal Roman numeral.  Since Roman numerals are written either as upper-case or as lower-case, and almost never as mixed or title-case, the solution splits the current graph accepting both upper and lower-case Roman numerals into two: one that will accept only upper-case and one that accepts only the lower-case numerals, leaving proper nous like `Li` denormalized.

Reply:
Yes, but you already have a graph that recognizes solely upper or lower case with your adding of the flag to the Roman generator, so I believe this is redundant no?

Reply:
`graph` points to the graph for lower-case, and `graph_upper` to upper-case Romans.  Without `graph_upper`, only lower-case Romans would be accepted.  Perhaps renaming `graph` to `graph_lower` would make it more intuitive?

Reply:
Sure, that would make it easier to follow. I believe you can also just place all the `uppers` and `lower` subgraphs together so you only need to build the graph once.

Reply:
Ok, let me try that.

Reply:
Done.